astore/server/auth/factory.go

```diff
@@ -24,6 +29,8 @@ func DefaultFlags() *Flags {
 
 func (f *Flags) Register(set kflags.FlagSet, prefix string) *Flags {
 	set.DurationVar(&f.TimeLimit, prefix+"time-limit", f.TimeLimit, "How long to wait at most for the user to complete authentication, before freeing resources")
+	set.StringVar(&f.Principals, prefix+"principals", f.Principals, "Allowed principals and the ability to auth,comma separated string")
```

Comment:
nit: change "auth,comma" to "auth, comma" (space)

Reply:
Also, what is the meaning of principals in this context? Can you provide an example? (in the help string above, not in comments on the PR)